packages/driver/src/cy/commands/querying/focused.ts

```diff
@@ -1,13 +1,13 @@
 import $dom from '../../../dom'
 
 export default (Commands, Cypress, cy, state) => {
-  Commands._addQuery('focused', function focused (options: Partial<Cypress.Loggable & Cypress.Timeoutable> = {}) {
+  Commands.addQuery('focused', function focused (options: Partial<Cypress.Loggable & Cypress.Timeoutable> = {}) {
     const log = options.log !== false && Cypress.log({ timeout: options.timeout })
 
     return () => {
       let $el = cy.getFocused()
 
-      log && log.set({
+      log && cy.state('current') === this && log.set({
```

Comment:
Under what circumstance would `cy.state('current')` not equal `this` when this is invoked?

Reply:
When re-running the query as part of a subjectChain later on. It solves a case like this:

```
cy.get('button').click()

setTimeout(() => {
  cy.$$('button').addClass('active').blur()
}, 1000)

cy.focused().eq(0).should('have.class', 'active')
```

At the same time the button gets the active class, it blurs, so the query+assertion ('the first focused element should have the active class') never pass at the same time. However, we want `.focused()` to show one element resolved in the command log: There *was* a focused element when it first passed, when we took the DOM snapshot, before moving onto .eq(0); the fact that we later retried the query shouldn't erase this fact.

(this is a contrived example, but equivalent stuff shows up 'in the wild' quite regularly. I hadn't noticed the case until Brian pointed it out during one of our demos)